loops-arrays-objects.js

```diff
@@ -131,5 +163,57 @@ const students = [
 // ]
 // ==========================================
 
-
-
+for (let i = 0; i <students.length ; i++) {
+    if (students[i].zipCode === '3513') {
+        (students[i].neighborhood = 'Pijlsweerd')
+    } else if  (students[i].zipCode === '3514') {
+        (students[i].neighborhood = 'Vogelenbuurt')
+    } else if (students[i].zipCode === '3512') {
+        (students[i].neighborhood = 'Binnenstad')
+    } else if (students[i].zipCode === '3531') {
+       (students[i].neighborhood = 'Lombok')
+    } else if (students[i].zipCode === '3572') {
+        (students[i].neighborhood = 'Wittevrouwen')
+    } else if (students[i].zipCode === '3581') {
+        (students[i].neighborhood = 'Oudwijk')
+    } else if (students[i].zipCode === '3583') {
+       (students[i].neighborhood = 'Schildersbuurt')
+    }
+}
+    console.log(students)
+
+
+console.log('**************************')
+console.log('onderdeel 1 en onderdeel 2 ')
+console.log('***************************')
+
+
+const zipCode = null //vraag: je wilt deze eigenlijk niet uitprinten
```

Comment:
Hoeft ook niet. Eigenlijk benaderen met: students[i].zipCode

students[0].zipCode ==> 3513.

if (students[i].zipCode === '3513') {
        (students[i].neighborhood = 'Pijlsweerd')

Maar zo te zien heb je dit wel goed in de smiezen.